eventfinder/src/eventcard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+    textAlign: 'left',
+    transition: '0.3s',
+    '&:hover':{
+      transform: 'scale(1.02)',
+      transition: '0.3s'
+    }
+  },
+  media: {
+    height: 140,
+  },
+});
+
+export default function EventCard() {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root}>
+      <CardActionArea>
+        <CardMedia
+          className={classes.media}
+          image="https://www.scitecheuropa.eu/wp-content/uploads/2019/07/ClaudioVentrella.jpg"
+          title="Contemplative Reptile"
+        />
+        <CardContent>
+          <Typography gutterBottom variant="h5" component="h2">
+            Good News Everyone! Futurama Trivia Night
+          </Typography>
+          <Typography variant="subtitle1" component="h4">
+           01/01/3021
```

Comment:
Sometimes events can span multiple days, or may only be during certain times, so we may want to approach this by using a `startDate` and `endDate`, and including a time. It could even be more elaborate by having different times for each day.

Maybe instead of one day, we would see something like this with variations:
```
01/01/3021 - 01/11/3021 2:30pm - 3:30pm
```
```
01/01/3021 2:30pm - 3:30pm
01/02/3021 1:30pm - 2:30pm
01/03/3021 2:30pm - 5:30pm
_See more_
```

We may choose to force them to be separate events. We can discuss it.


This is not something we have to solve in this PR, but if we choose to wait, we should capture it in an issue.

Reply:
Captured in an issue